server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -678,9 +678,20 @@ public void run()
                 .withEmitter(emitter)
                 .withBalancerStrategy(balancerStrategy)
                 .build();
+
+        boolean coordinationPaused = getDynamicConfigs().getPauseCoordination();
+        if (coordinationPaused && coordLeaderSelector.isLeader()) {
```

Comment:
I think this `if` clause should check `startingLeaderCounter == coordLeaderSelector.localTerm()` as well as in the below `if` clause. Or maybe, this log can be just printed when the dynamic configuration is updated. Also we are trying to reduce the amount of logs. Do you think it's worth to be the info level? Or can it be the debug level?

Reply:
I agree that this can be flipped to a debug log. It was nice to have it as info for our initial testing to make sure everything was squared away. I will swap to debug and add to the conditional check to cover the case you mentioned. Now that we know everything works, admins will just need to reference the dynamic config instead of logs for clarity on the state of this (which seems like better admin practice anyways).